config/policy-controller/default/kustomization.yaml

```diff
@@ -0,0 +1,16 @@
+
+# Value of this field is prepended to the
+# names of all resources, e.g. a deployment named
+# "wordpress" becomes "alices-wordpress".
+# Note that it should also match with the prefix (text before '-') of the namespace
+# field above.
+namePrefix: kuadrant-
```

Comment:
Does "kuadrant" make sense a prefix? This gets added to the names of all resources, so when you are including it from other kustomizations which also use a namePrefix, it prepends them so you end up with the likes of this in kuadrant-operator `kuadrant-operator-kuadrant-policy-controller-5bbb7545bf-vd5jw`.

Maybe we don't need a prefix here since it's always included by something else that is adding one?